app/src/main/java/org/oppia/android/app/onboarding/OnboardingFragmentPresenter.kt

```diff
@@ -112,12 +118,44 @@ class OnboardingFragmentPresenter @Inject constructor(
       .registerViewDataBinder(
         viewType = ViewType.ONBOARDING_FINAL_SLIDE,
         inflateDataBinding = OnboardingSlideFinalBinding::inflate,
-        setViewModel = OnboardingSlideFinalBinding::setViewModel,
+        setViewModel = this::bindOnboardingSlideFinal,
         transformViewModel = { it as OnboardingSlideFinalViewModel }
       )
       .build()
   }
 
+  private fun bindOnboardingSlideFinal(
+    binding: OnboardingSlideFinalBinding,
+    model: OnboardingSlideFinalViewModel
+  ) {
+    binding.viewModel = model
+    setTermsAndCondition(binding)
+  }
+
+  private fun setTermsAndCondition(binding: OnboardingSlideFinalBinding) {
```

Comment:
Is this helper actually needed? It seems unnecessary since it's just called in the binding method right above.

Reply:
To keep it clean or make it short I added a helper here. Removed it and added in  the binding method.